Creating initial documentation

Aeywoo opened the pull request:
Creating a README.md with documentation on the extension.

Aeywoo committed:
Creating initial documentation

Creating a README.md with documentation on the extension.

Aeywoo committed:
Creating initial documentation

Better grammar.

Aeywoo committed:
Creating initial documentation

This sounds better.

Aeywoo committed:
Creating initial documentation

🗯️

G-DR-WIKI commented on README.md:
```diff
@@ -0,0 +1,43 @@
+- It adds an area for Discourse profile elements to the top of every page related to user actions, this includes user pages (`User:Example`), user talk pages (`User_talk:Example`) and user contribution pages (`Special:Contributions/Example`).
+- It adds a users Discourse avatar, their wiki edit count and the amount of Discourse posts they've made, along with related user page links at the bottom of the Discourse profile element, these links go to each persons `Special:Contributions` entry, `User_talk` page and to their Discourse posts page.
+- Has the option to combine user groups from Discourse and MediaWiki together, eliminating the need to manually promote staff members through Discourse and MediaWiki.
+
```
Missing what the extension does to replace talk pages with discourse:
- Maintenance script to sync Discourse tags with qualifying (mainspace, no sub-pages) articles on the wiki
- Redirects an article's talk link to the relevant tag on Discourse.
- Shows a list of relevant post previews at the bottom of qualifying articles.

Aeywoo committed:
Creating initial documentation

Adding more to the features section.

Aeywoo committed:
Creating initial documentation

Updated wording a bit more.

Aeywoo committed:
Creating initial documentation

Updating some formatting and added a new column to the configuration options table and removed an unnecessary sub-header.
